src/components/marquee-wizard/marquee-wizard.component.ts

```diff
@@ -45,32 +43,24 @@ export class MarqueeWizardComponent extends WizardComponent implements OnDestroy
      */
     _isInitialised: boolean = false;
 
-    /** Unsubscribe from all subscriptions when component is destroyed */
-    private _onDestroyed = new Subject<void>();
-
     get isTemplate(): boolean {
         return this.description && this.description instanceof TemplateRef;
     }
 
-    constructor(marqueeWizardService: MarqueeWizardService,
+    constructor(protected wizardService: WizardService,
```

Comment:
I don't think this needs to be `protected` here, as we are passing it to the base class which has it as a protected member. 
Just this should do:

```typescript
constructor(wizardService: WizardService,
```